src/EditorFeatures/TestUtilities/SignatureHelp/SignatureHelpTestItem.cs

```diff
@@ -37,20 +37,28 @@ public struct SignatureHelpTestItem
         /// </summary>
         public readonly string PrettyPrintedSignature;
 
+        /// <summary>
+        /// Whether this item is expected to be selected.
+        /// Note: If no item is expected to be selected, the verification of the actual selected item is skipped.
+        /// </summary>
+        public readonly bool IsSelected;
```

Comment:
:bulb: Could make this a `bool?` instead which defaults to `null` and verification is only performed when set

Reply:
I gave it a try, but didn't find that to help. 
I found there is not much value to distinguishing `null` and `false` with that approach, and setting `false` in a number of expected items was a cost with little value.